doc/releases.md

```diff
@@ -0,0 +1,60 @@
+# OpenConfig Release Versioning
+
+## Rationale
+
+As the OpenConfig project matures and is adopted by more implementors and
+network operators, model changes are becoming increasingly frequent. This has
+made identifying compatible and well-known sets of models more difficult.
+Furthermore, uncontrolled and frequent breaking changes can create excessive
+burden on implementors and operators alike.
+
+This revision proposal aims to address these problems by introducing
+[semantic versioning](https://semver.org/) to the set of OpenConfig models as a
+whole along with some OpenConfig-specific guidelines. As a side note, these
+releases are compatible with the notion of
+[YANG release bundles](https://github.com/openconfig/public/blob/master/release/models/catalog/openconfig-module-catalog.yang).
+
+## Policy
+
+### Basic Guidelines
+
+1.  A regular release of https://github.com/openconfig/public containing a set
+    of compatible models (consisting of the entire set of models within the
+    `openconfig/public` repo) is created roughly every quarter. The tag is named
+    vx.y.z (e.g. v1.2.0) following
+    [semantic versioning rules](https://semver.org/). A major, minor, or patch
+    version increment is possible at each release.
+2.  Any backwards-incompatible change for a pre-1.0.0 model does NOT on its own
+    necessitate a major revision change for the overall models repository.
+3.  [Patch releases](https://semver.org/#spec-item-6) may be created at any time
+    for backwards-compatible bug fixes.
+4.  [Pre-releases](https://semver.org/#spec-item-9) may be created at anytime to
+    quickly introduce new changes to the models. These are not intended to be
+    long-term, stable releases -- they should be replaced with the next regular
+    release that encompasses these changes as soon as it becomes available.
+5.  Backwards-incompatible model changes affecting a feature that has reasonable
+    functional test coverage via
+    [OpenConfig featureprofiles](https://github.com/openconfig/featureprofiles/))
+    or implemented on a device SHOULD be made infrequently. The OpenConfig
+    working group will create backwards-incompatible releases periodically by
+    considering both velocity and maintenance cost implications.
```

Comment:
Added example.

Reply:
The example is good, but it might be good to indicate (maybe up in point 1 where we mention a release per quarter) that the non-backwards-compatible changes (e.g. major digit change) should be collected and published at a less frequent cadence (e.g. perhaps once per year) than the backwards-compatible changes.  As a high level philosophy (where there may also be occasional exceptions of course depending on the expected impact to the community, etc).

Reply:
added sentence that major releases should be less frequent